content/md/first-steps/manage-your-attributes.md

```diff
@@ -32,7 +32,7 @@ In addition to these required attributes, you can select the following propertie
     - **Read only** (Yes/No) **_(Enterprise Edition only)_**: to prevent this attribute from being edited through the UI, it will only be updated through imports, API or rules
     - **Usable in grid** (Yes/No): if you want to filter on this attribute in the product's grids or use it as a column.
```

Comment:
```suggestion
    - **Usable in grid** (Yes/No): if you want to filter on this attribute in the product grid or use it as a column.
```

Reply:
in the blue non-numbered above line 32, the last word should be "properties"

Reply:
in the blue non-numbered line above line 79, I suggest "Each attribute has specific properties to add (or "to provide") more control over its value" or "...over attribute values."